### PR TITLE
Azure storage queue sink

### LINKED ISSUE #138: Create Azure Storage Queue Sink Kamelet

### PR DESCRIPTION
**Pull request by oscerd**
Fixes #138

### CODE REVIEW

File: azure-storage-queue-sink.kamelet.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    camel.apache.org/kamelet.type: "sink"
+spec:
+  definition:
+    title: "Azure Storage Queue Sink"
+    description: |-
+      Send Messages to Azure Storage queues.
+
+      The Kamelet is able to understand the following headers to be set:
+
+      - `expiration` / `ce-expiration`: as the time to live of the message in the queue.
```
Comment (nicolaferraro) [inline azure-storage-queue-sink.kamelet.yaml:18]: Can you describe the format expected for this header?

**Comment by oscerd** [comment]
Yeah, it's a duration. But i will copy the catalog description

**Review by nicolaferraro — CHANGES_REQUESTED** [review]
Only a comment about the expiration header

File: azure-storage-queue-sink.kamelet.yaml (hunk 1 above)
Comment (oscerd) [reply azure-storage-queue-sink.kamelet.yaml:18]: done

**Review by nicolaferraro — APPROVED** [review]

**Pull request merged**